### PR TITLE
[v7r1] Several fixes for scripts

### PR DESCRIPTION
**Pull request by fstagni**
The `Script.showHelp(exitCode=0)` function is already exiting with `DIRAC.exit(exitCode)`. Most of scripts were ignoring it, so this has been fixed. Several files changed, but with simple fixes.

BEGINRELEASENOTES

*WMS
FIX: PilotStatusAgent was crashing due to a bug

*Test
FIX: fix for main RSS system test

ENDRELEASENOTES

### CODE REVIEW

File: Interfaces/API/DiracAdmin.py [hunk 1]
```diff
@@ -253,7 +253,7 @@ def getSiteMaskLogging(self, site=None, printOutput=False):
 
       sitesLogging = result['Value']
       if isinstance(sitesLogging, dict):
-        for siteName, tupleList in sitesLogging.iteritems():
+	for siteName, tupleList in sitesLogging.items():  # can be an iterator
```
Comment (chrisburr) [inline Interfaces/API/DiracAdmin.py:256]: I guess this indentation should have been:
```suggestion
	  for siteName, tupleList in sitesLogging.items():  # can be an iterator
```

File: Interfaces/API/DiracAdmin.py [hunk 2]
```diff
@@ -262,9 +262,11 @@ def getSiteMaskLogging(self, site=None, printOutput=False):
             gLogger.notice(stup)
           gLogger.notice(' ')
       elif isinstance(sitesLogging, list):
-        result = [(sl[1], sl[3], sl[4]) for sl in sitesLogging]
+	sitesLoggingList = [(sl[1], sl[3], sl[4]) for sl in sitesLogging]
+	for siteLog in sitesLoggingList:
+	  gLogger.notice(siteLog)
```
Comment (chrisburr) [inline Interfaces/API/DiracAdmin.py:267]: ?
```suggestion
	  sitesLoggingList = [(sl[1], sl[3], sl[4]) for sl in sitesLogging]
	  for siteLog in sitesLoggingList:
	    gLogger.notice(siteLog)
```

File: Interfaces/API/DiracAdmin.py [hunk 3]
```diff
@@ -262,9 +262,11 @@ def getSiteMaskLogging(self, site=None, printOutput=False):
...
 
-    return result
+    return S_OK()
```
Comment (chrisburr) [inline Interfaces/API/DiracAdmin.py:269]: Is this right? I haven't looked properly but I was expecting it to return the result of line 241 or 243.

File: Interfaces/API/DiracAdmin.py (hunk 2 above)
Comment (fstagni) [reply Interfaces/API/DiracAdmin.py:267]: I introduced by mistake some tabs instead of spaces...

File: Interfaces/API/DiracAdmin.py (hunk 3 above)
Comment (fstagni) [reply Interfaces/API/DiracAdmin.py:269]: That's returned at line 246 if it's `S_ERROR`, otherwise if it reaches here it's only the printout.

**Pull request merged**